src/api/app/views/webui2/webui/users/tasks/index.html.haml

```diff
@@ -0,0 +1,87 @@
+:ruby
+  @pagetitle = "Tasks for #{User.current}"
+  @layouttype = 'custom'
+  @crumb_list = [@pagetitle]
+
+.container
+  .card#reviews
+    .bg-light
+      %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+        %li.nav-item
+          - cache "#{User.current.cache_key}-reviews-in" do
+            %a{ href: '#reviews-in', title: "Requests that #{User.current.login} has to review", class: 'nav-link text-nowrap active' }
```

Comment:
Let's put the CSS classes directly on the `%a`, so `%a.nav-link.text-nowrap.active`